capa/features/extractors/cape/file.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (C) 2020 Mandiant, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+import logging
+from typing import Any, Dict, List, Tuple, Iterator
+
+from capa.features.common import Feature, String, Registry, Filename, Mutex
+from capa.features.file import Section, Import, Export, FunctionName
+from capa.features.address import Address, AbsoluteVirtualAddress, NO_ADDRESS
+
+
+logger = logging.getLogger(__name__)
+
+
+def extract_import_names(static: Dict) -> Iterator[Tuple[Feature, Address]]:
+    """
+    extract the names of imported library files, for example: USER32.dll
+    """
+    for library in static["imports"]:
+        name, address = library["name"], int(library["virtual_address"], 16)
+        yield Import(name), address
```

Comment:
this is cool!

Reply:
consider using helper here in order to generate all the combinations:
https://github.com/mandiant/capa/blob/7ef78fdbce3959529eed5d8bd96b67348838f6e7/capa/features/extractors/helpers.py#L44